ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -78,4 +78,20 @@
 			<Link>www\%(RecursiveDir)%(Filename)%(Extension)</Link>
 		</Content>
 	</ItemGroup>
+
+	<ItemGroup Condition="Exists("overlay\os\$(ASFVariant.Split('-')[0])")">
+		<Content Include="overlay\os\$(ASFVariant.Split('-')[0])\**\*.*">
```

Comment:
If you want it to be operating system specific, we should use `RuntimeIdentifier` instead of `ASFVariant`